quickstarts/hello-rules-multi-module/README.md

```diff
@@ -0,0 +1,120 @@
+## Red Hat Process Automation Manager KIE Server decisions Quickstart - Multi module
+
+This quickstart is intended to be used with the [RHPAM Kie Server](https://github.com/jboss-container-images/rhpam-7-openshift-image/tree/main/kieserver) image.
+
+## How to use it?
+
+To deploy the Hello Rules demo you can use the [rhdm713-prod-immutable-kieserver](https://github.com/jboss-container-images/rhpam-7-openshift-image/blob/main/templates/rhdm713-prod-immutable-kieserver.yaml)
```

Comment:
Since we renamed every occurrence of RHDM with RHPAM, do we want to do something similar also for the templates (i.e. rhpam-7.13-dm-...)?

Reply:
at this moment the naming will still the same.